app/assets/javascripts/angular-libs/guides/new/guides.new.chooseCrop.directive.js

```diff
@@ -0,0 +1,34 @@
+openFarmApp.directive('guidesChooseCrop' ,[
+  function guidesChooseCrop() {
+    return {
+      restrict: 'A',
+      scope: {
+        texts: '=',
+        guide: '=',
+      },
+      controller: ['$scope', '$element', '$attrs',
+        function ($scope, $element, $attrs) {
+          //Gets fired when user selects dropdown.
+          $scope.cropSelected = function ($item, $model, $label) {
+            $scope.guide.crop = $item;
+            $scope.crop_not_found = false;
+            $scope.guide.crop.description = '';
+          };
+
+          //Gets fired when user resets their selection.
+          $scope.clearCropSelection = function ($item, $model, $label) {
```

Comment:
'$label' is defined but never used.<br>'$model' is defined but never used.<br>'$item' is defined but never used.